src/viam/sdk/module/service.hpp

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include "viam/sdk/registry/registry.hpp"
```

Comment:
Can we move this include to be more consistent with our include sorting elsewhere?